### PR TITLE
Features/add replay stream twins

### PR DESCRIPTION
**Pull request by p3root**
And remove unused properties

### CODE REVIEW

File: DTDL/replay_value_source.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "extends": "dtmi:io:tributech:source:base;1",
+    "displayName": "Replay Source",
+    "contents": [
+        {
+            "@type": "Property",
+            "name": "RowValueMetadataId",
+            "schema": "string",
+            "writable": true,
+            "displayName": "RowValueMetadataId",
+            "description": "The value MetadataId for the row json object"
```
Comment (mayrmax) [inline DTDL/replay_value_source.json:14]: i dont get this description

File: DTDL/replay_value_source.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        },
+        {
+            "@type": "Property",
+            "name": "FileName",
+            "schema": "string",
+            "writable": true,
+            "displayName": "Frequency",
```
Comment (mayrmax) [inline DTDL/replay_value_source.json:21]: File Name

File: DTDL/replay_value_source.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            "description": "Line by line interval in ms"
```
Comment (mayrmax) [inline DTDL/replay_value_source.json:22]: The name of the file that should be loaded.

Or something like that.

File: DTDL/replay_value_source.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        },
+        {
+            "@type": "Property",
+            "name": "RestartAutomatically",
+            "schema": "boolean",
+            "writable": true,
+            "displayName": "Frequency",
```
Comment (mayrmax) [inline DTDL/replay_value_source.json:37]: Restart Automatically

Comment (mayrmax) [reply DTDL/replay_value_source.json:37]: i think a better name for the property and display name would be even better. something like infinite replay

File: DTDL/replay_value_source.json [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+            "description": "Line by line interval in ms"
```
Comment (mayrmax) [inline DTDL/replay_value_source.json:38]: Infinitely replay the file.

File: DTDL/replay_value_source.json [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+        },
+        {
+            "@type": "Relationship",
+            "name": "Streams",
+            "minMultiplicity": 0,
+            "maxMultiplicity": 10,
+            "target": "dtmi:io:tributech:stream:replay;1",
+            "displayName": "Streams",
+            "description": "Contains all Simualted streams the given source."
```
Comment (mayrmax) [inline DTDL/replay_value_source.json:47]: Link to all replay streams of this source.

File: DTDL/replay_value_source.json (hunk 1 above)
Comment (mayrmax) [inline DTDL/replay_value_source.json:14]: better description please

File: DTDL/replay_value_source.json (hunk 2 above)
Comment (mayrmax) [inline DTDL/replay_value_source.json:21]: File Name

File: DTDL/replay_value_source.json (hunk 3 above)
Comment (mayrmax) [inline DTDL/replay_value_source.json:22]: The name of the file used for the replay source.

File: DTDL/replay_value_source.json [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+            "displayName": "Frequency",
+            "description": "Line by line interval in ms"
+        },
+        {
+            "@type": "Property",
+            "name": "Frequency",
+            "schema": "double",
+            "writable": true,
+            "displayName": "Frequency",
+            "description": "Line by line interval in ms"
```
Comment (mayrmax) [inline DTDL/replay_value_source.json:30]: Interval used to replay values read line-by-line from the file in ms.

File: DTDL/replay_value_source.json [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+        },
+        {
+            "@type": "Property",
+            "name": "RestartAutomatically",
```
Comment (mayrmax) [inline DTDL/replay_value_source.json:34]: InfiniteLoop

File: DTDL/replay_value_source.json (hunk 4 above)
Comment (mayrmax) [inline DTDL/replay_value_source.json:37]: Infinite Loop

File: DTDL/replay_value_source.json (hunk 5 above)
Comment (mayrmax) [inline DTDL/replay_value_source.json:38]: Indicates if file content should be looped infinitely.

File: DTDL/replay_value_source.json (hunk 6 above)
Comment (mayrmax) [inline DTDL/replay_value_source.json:47]: Link to all replay streams of this source.

File: DTDL/replay_value_stream.json [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+    "displayName": "Replay Stream",
+    "extends": "dtmi:io:tributech:stream:base;1",
+    "contents": [
+        {
+            "@type": "Property",
+            "name": "ColumnName",
+            "schema": "string",
+            "writable": true,
+            "displayName": "Column name",
+            "description": "Header column name inside the csv file"
```
Comment (mayrmax) [inline DTDL/replay_value_stream.json:14]: Name of the column header used for this value stream.

**Comment by mayrmax** [comment]
Can you add a demo for the Replay source please

**Review by simonpfeifhofer — APPROVED** [review]

**Pull request merged**